sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/SessionRecords/NotificationHookLiveTests/UpdateWebNotificationHookWithMinimumSetupAndNewInstance.json

```diff
@@ -8,8 +8,11 @@
         "Content-Length": "98",
         "Content-Type": "application/json",
         "Ocp-Apim-Subscription-Key": "Sanitized",
-        "traceparent": "00-0a4b6c4dbb0bab42a9c90f0a2a0b3d69-4e8ec5da11475d43-00",
```

Comment:
what changed from the request/response that caused the re-recordings?

Reply:
Previously, we were forcing users to set name/endpoint or name/emails when creating a hook. This is a requirement for the Create method, but not for the Update one. Since the constructor is now parameterless and properties are settable, we're not required to set these properties again when updating, so I made a small clean up to remove them where not required.